__tests__/parse.spec.js

```diff
@@ -230,7 +230,7 @@ index 123..456 789
 +
  Therefore let there always be non-being,
 	so we may see their subtlety,
-And let there always be being,
+ And let there always be being,
```

Comment:
I think space was missing here, not sure if that was intentional.

Reply:
ok